de.dlr.sc.virsat.team.ui/src/de/dlr/sc/virsat/team/ui/handler/git/GitUpdateHandler.java

```diff
@@ -43,7 +43,7 @@ protected void doUpdate(IProject project, IProgressMonitor monitor) throws Excep
 		
 		Repository gitRepository = RepositoryMapping.getMapping(project).getRepository();
 
-		// Build the dialog in the UI thread
+		// Build the dialog in the UI threads

```

Comment:
I think the comment is right, there is just one UI thread :)

Reply:
Yes, i think too.